src/app/pages/settings/containers/settings-page.component.ts

```diff
@@ -179,7 +190,7 @@ export class SettingsPageComponent {
           const promises = []
           if (selected.includes(ResetOption.ENROLMENT))
             promises.push(this.settingsService.resetAuth())
-          if (selected.includes(ResetOption.CONFIG))
+          else if (selected.includes(ResetOption.CONFIG))
             promises.push(this.settingsService.resetConfig())
           if (selected.includes(ResetOption.CACHE))
             promises.push(this.settingsService.resetCache())
```

Comment:
should this also be in else if? because if enrolment is reset the cache is reset too?